### DISCUSSION TITLE
Download size capped at 144p

### DISCUSSION THREAD

**Discussion by gridsleep** [aclap-dev/video-downloadhelper#182] [Help and Q&A]
### Your question

I just noticed that any selected file will download at no larger than 144p (240p). I haven't downloaded in at least a month. Is this an error in my machine or has Youtube cramped the software's style? Windows 10, Firefox

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 2b98742cc6c6462b
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\gridsleep",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by GrampaWildWilly** [comment]
I downloaded something off YouTube earlier & it worked fine.  I'll tell you what it is if you tell me what you're having trouble with.

**Comment by gridsleep** [comment]
Any video I selected to download is limited to 240p. I just tried again and the program is working normally again. The first video is available at 1080p. I guess the problem may have been a temporary server error.

**Reply by GrampaWildWilly** [reply]
I have found on occasion if I stumble upon something that was just posted, the best resolution on offer is not very good.  But if I wait a while, which can be anywhere from 5 minutes to half an hour or so, the rest of the resolutions show up.  So you might take a quick look at the post time next time you hit something like this.
